Framework/src/TaskRunner.cxx

```diff
@@ -339,15 +350,15 @@ void TaskRunner::startOfActivity()
   mTimerTotalDurationActivity.reset();
   mTotalNumberObjectsPublished = 0;
 
-  Activity activity(mConfigFile->get<int>("qc.config.Activity.number"),
+  Activity activity(mRunNumber,
```

Comment:
If we don't respect "qc.config.Activity.number" anymore, then we should delete it in config files. Otherwise, we could respect it if mRunNumber == 0.

Reply:
I agree, will do it